homeassistant/components/api.py

```diff
@@ -79,6 +80,7 @@ def get(self, request):
         if restrict:
             restrict = restrict.split(',') + [EVENT_HOMEASSISTANT_STOP]
 
+        @asyncio.coroutine
```

Comment:
If we mark it as a co-routine, shouldn't we also then register it using `hass.bus.listen_async` instead of `hass.bus.listen` ?